pkg/kapp/resourcesmisc/kappctrl_k14s_io_v1alpha1_app.go

```diff
@@ -64,10 +64,5 @@ func (s KappctrlK14sIoV1alpha1App) IsDoneApplying() DoneApplyState {
 		}
 	}
 
-	deletingRes := NewDeleting(s.resource)
```

Comment:
note that the reason why this was set up this way (not saying it's ideal) is because above (line 61) is looking at delete conditions.